project-base/src/Controller/Front/NewsletterController.php

```diff
@@ -53,12 +64,43 @@ public function subscribeEmailAction(Request $request)
         $form = $this->createSubscriptionForm();
         $form->handleRequest($request);
 
-        if ($form->isSubmitted() && $form->isValid()) {
-            $email = $form->getData()['email'];
-            $this->newsletterFacade->addSubscribedEmail($email, $this->domain->getId());
+        if ($form->isSubmitted()) {
+            $this->formTimeProvider->generateFormTime($form->getName());
```

Comment:
is it necessary to generate formtime manually? I thought it's handled by listener on form pre-submit event

Reply:
27223429b561378717bd3b0702a3ff030822cb8d